pubspec.yaml

```
@@ -1,21 +1,20 @@
 name: reorderable_carousel
 description: A carousel in that allows one to reorder the contents of it via drag and drop 
-version: 0.3.2+1
+version: 0.4.0
 homepage: https://github.com/TNorbury/reorderable_carousel
 repository: https://github.com/TNorbury/reorderable_carousel
 issue_tracker: https://github.com/TNorbury/reorderable_carousel/issues
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=2.0.0"
+  sdk: ">=2.17.0 <3.0.0"
```

This needs to stay in here to indicate that Flutter is required in order to use this package
```suggestion
  sdk: ">=2.17.0 <3.0.0"
  flutter: ">=2.0.0"
```

---

I was incorrect here, it can be left as is.